datajoint/external.py

```diff
@@ -87,24 +100,38 @@ def get(self, hash):
         """
         store = hash[STORE_HASH_LENGTH:]
```

Comment:
Would you do me a favor and replace the variable name `hash` with something else? `hash` is the name of a built-in Python function and I much rather not have it's name used as variable.

Reply:
`hash_`? `hashed`? `blob_hash`?

Reply:
Too late now, but probably `blob_hash`